pkg/proxy/proxy.go

```diff
@@ -5,18 +5,20 @@ import (
 	"net"
 )
 
+// Proxy defines the behavior of a proxy.
```

Comment:
This is a little a tough to track down. After some discussion with @icecrime, we thought it might be better to overlook some inaccurate comments in the interest of enabling golint across the board. A quick glance at this PR with those goggles shows no problems with this PR as is.

We can go back and do the archaeological work as we make changes to the code base in the future.